ci/redmine_bugzilla.py

```diff
@@ -0,0 +1,153 @@
+"""
+This module does a few things:
+
+1) Verify that all Bugzilla bugs which point to a Redmine issues also have
+Redmine issues pointing back to Bugzilla bugs. E-mail a list of people if
+only a uni-directional relationship exists so they can be fixed manually. The
+EMAIL_RECEIVERS constant specifies the recipient list.
+
+2) For all Redmine issues that track Bugzilla bugs, ensure that the
+Bugzilla external tracker description, status, and priority all mirror the
+Redmine issue state.
+
+Requirements:
+The Redmine and Bugzilla credentials are expected from to be read from a
+file in ini format. See the example below. The location of this file is set
+by the REDMINE_BUGZILLA_CONF environment variable.
+
+<SNIP>
+[bugzilla]
+username = XXXX
+password = XXXX
+
+[redmine]
+key = XXXX
+</SNIP>
+"""
+
+
+import ConfigParser
+import os
+
+from bugzilla.rhbugzilla import RHBugzilla
```

Comment:
It is in the bash script in the yaml file. It's the line:

`sudo pip install --upgrade git+https://github.com/bmbouter/python-bugzilla.git@expand-external-tracker-support`